index.js

```diff
@@ -313,38 +314,35 @@ function openMergeRequests(options) {
     }
 
     getURLOfRemote(remote).then(function (remoteURL) {
-      var projectName = remoteURL.match(regexParseProjectName)[2];
+      getUser(options.assignee).then(function (assignee) {
+        var projectName = remoteURL.match(regexParseProjectName)[2];
 
-      var queryStringToAppend = "?";
-      if(options.state){
-        queryStringToAppend += "state="  + options.state + "&";
-      }
+        var query = '?';
 
+        if (options.state) {
+          query += 'state=' + options.state + '&';
+        }
 
-      getUserId(options).then(function(userId){
-        if(userId){
-          queryStringToAppend += "assignee_id=" + userId + "&";
+        if (assignee) {
+          query += 'assignee_username=' + assignee.username + '&';
```

Comment:
@stevelacey 
I dont mind everything else but lets keep this `assignee_id ` as earlier. I had used assignee_id after looking at my merge request url in gitlab version 8.8. 

We just recently upgraded to 10.1 and I see that it uses username now instead of assignee_id like you said. For keeping backward compatibility with previous version I will want to keep it assignee_id. It works in 10.1 as well. I checked.

Reply:
@vishwanatharondekar reverted 👍